impl/src/main/java/com/pavlovmedia/oss/osgi/gelf/impl/PavlovGelfTcpTransporter.java

```diff
@@ -70,7 +71,12 @@
             name = PavlovGelfTcpTransporter.GRAYLOG_THREAD_POOL_SIZE,
             intValue = PavlovGelfTcpTransporter.GRAYLOG_THREAD_POOL_SIZE_DEFAULT,
             label = "thread Pool size",
-            description = "message processing thread pool size (minimum 10)") })
+            description = "message processing thread pool size (minimum 10)"),
+        @Property(
+                name=PavlovGelfTcpTransporter.GRAYLOG_HOSTNAME, 
+                label="Source hostname", 
+                description="If non-empty, this will be used as the hostname in logging messages")
```

Comment:
If empty, will attempt to use the fully qualified domain name for this IP address